airbyte-integrations/connectors/source-zenloop/integration_tests/catalog.json

```diff
@@ -25,6 +25,12 @@
       "source_defined_cursor": true,
       "default_cursor_field": "test",
       "json_schema": {}
+    },
+    {
+      "name": "properties",
+      "supported_sync_modes": ["full_refresh"],
+      "source_defined_cursor": true,
+      "json_schema": {}
```

Comment:
Set value of property `json_schema` to contents of `airbyte-integrations/connectors/source-zenloop/source_zenloop/schemas/properties.json`.

Reply:
set